src/Trinity.Core/Trinity/Configuration/TrinityConfig.cs

```diff
@@ -62,7 +62,14 @@ public unsafe static partial class TrinityConfig
         static TrinityConfig()
         {
             GetConfigurationInstances().ToList();
-            LoadTrinityConfig(false);
+            try
+            {
+                LoadTrinityConfig();
+            }
+            catch
+            {
+                Log.WriteLine(LogLevel.Error, "Failure to load config file, the default configuration takes effect");
```

Comment:
I have a concern that this message will be displayed multiple times by different classes. please verify.